cultcargo/quartical.yml

```diff
@@ -43,4 +43,31 @@ cabs:
               zarr that will be created, e.g. path/to/dir.
       column:
         dtype: str
-        info: Name of column to be backed up.    
+        info: Name of column to be backed up.
+
+  quartical-restore:
+    name: quartical-restore
+    image:
+      _use: vars.cult-cargo.images
+      name: quartical
+      # version: # Optionally.
+    command: goquartical-restore
+    info: QuartiCal column restoring script
+
+    policies:
+      positional: true
+
+    inputs:
+      zarr_path:
+        dtype: Directory
+        must_exist: true
+        info: Path to backup zarr column e.g. path/to/dir/20211201-154457-foo.MS-FLAG.bkp.qc.
+              Also accepts valid s3 urls.
+      ms_path:
+        dtype: Directory
```

Comment:
@JSKenyon same here.

Reply:
I have updated the backup and restore cabs - could you please give them a once over to check that I haven't done something unholy @o-smirnov?